### PR TITLE
Adding .travis.yml.

### LINKED ISSUE #5: Add .travis.yml file

### PR DESCRIPTION
**Pull request by ldko**
Adds a .travis.yml file. Will close #5 .

### CODE REVIEW

**Comment by ldko** [comment]
ping @somexpert @damonkelley

**Comment by damonkelley** [comment]
:+1: This looks good.

We are going to need to update `runtests.py` though. This build should be failing. The test script is not returning an accurate exit code upon failure. But you are probably already aware of this. :wink:

**Comment by somexpert** [comment]
:thumbsup:

**Comment by ldko** [comment]
yay fail, fail, fail now! Just like we hoped. :) you want to check the last commit @somexpert @damonkelley https://github.com/unt-libraries/pyuntl/commit/c7e0335d1e809d94bfcb8682da76938bc81bb91e

File: runtests.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env python
+import sys
 import unittest
-from tests import record, etd, field, reader, writer, pyuntl_test, dublincore, \
-    highwire
+
+from tests import (record, etd, field, reader, writer,
+                   pyuntl_test, dublincore, highwire)
```
Comment (damonkelley) [inline runtests.py]: :raised_hands:

**Comment by damonkelley** [comment]
:+1:

**Comment by somexpert** [comment]
:thumbsup:

**Pull request merged**